awx/main/models/credential/__init__.py

```diff
@@ -933,6 +933,10 @@ def openstack(cls):
                                            'It is only needed for Keystone v3 authentication '
                                            'URLs. Refer to Ansible Tower documentation for '
                                            'common scenarios.')
+            }, {
+                'id': 'no_verify_ssl',
```

Comment:
@ryanpetrello this is a little different from other sources where we tend to have `verify_ssl`, what do you think?

Reply:
@AlanCoding @ryanpetrello Thank you very much for the review!
So I will modify it from `no_verify_ssl` to `verify_ssl` as the same with Tower credential type and set the default value to True.